script/script.js

```diff
@@ -34,25 +40,14 @@ const setConfigItem = (path, value, obj) => {
     return obj;
 };
 
-for (const supportedFileExtension of supportedFileExtensions) {
-    // Load rules
-    const ruleFilePath = "rules/" + supportedFileExtension + ".yaml";
-    console.log("Loading " + ruleFilePath);
-    fetch(ruleFilePath)
-        .then(response => response.text())
-        .then((data) => {
-            localStorage.setItem(supportedFileExtension, data);
-        }).catch(error => console.error(error));
-}
-
 async function sanitizeContent(file) {
     document.getElementById("display_panel").hidden = false;
     showSpinner();
     const fileName = file.name;
     const fileNameParts = file.name.split('.');
     const fileExtension = fileNameParts.pop();
 
-    if (!supportedFileExtensions.includes(fileExtension)) {
+    if (!config["SupportedFileExtensions"].includes(fileExtension)) {
```

Comment:
Should we check if config's data was loaded properly?

Reply:
Can you elaborate?

Reply:
Let's say the config wasn't loaded properly, and the user chooses a file to sanitize, should we add a check here to make sure that `config["SupportedFileExtensions"]` doesn't return undefined?

Reply:
Good catch. We definitely want to handle that gracefully. I've added a default config value to fallback to in case of such errors after alerting/logging it.